README.rst

```diff
@@ -1,5 +1,5 @@
-.. image:: doc/source/images/pymechanical-logo-light.png
-   :alt: PyMechanical
+.. image:: https://raw.githubusercontent.com/ansys/pymechanical/main/doc/source/_static/logo/pymechanical-logo-light.png
```

Comment:
I wouldn't use the public facing URL - use the local path

Reply:
okay. but if we use the local path, then it wont show up in pypi page ➡️  https://pypi.org/project/ansys-mechanical-core/

if that's okay , I can update with local path.

Reply:
Ah got it. Okay, let's leave it like this then